frontend/app/templates/components/transportation/study-area-map.hbs

```diff
@@ -23,24 +23,51 @@
         paint=(hash fill-opacity=0)
       }} as |layer|
     >
-      <Mapbox::LayerHoverCard
+      <Mapbox::CurrentMouseoverFeatures
```

Comment:
can we consolidate the two `CurrentMouseoverFeatures` into one, and just add the 
```
@onFeatures={{this.setFirstHoveredFeatureId}}
``` 
from the other one in here? Doesn't feel overload-y to me. We're just handling all the "hover" actions/interactions in one component.

Reply:
Oh yeah! And I wouldn't even need to refactor `setFirstHoveredFeatureId` to do anything extra, since the HoverCard makes use of the yield!